### ISSUE TITLE
Scalar Equation is incorrect ? Documentation

### ISSUE THREAD

**Issue by ArEnSc** [tensorflow/tensorflow#78]
```
http://tensorflow.org/tutorials/mnist/beginners/index.md#the_mnist_data

https://upload.wikimedia.org/math/7/3/f/73fc7ef1bf6f6822115c41cff58535e1.png

http://api.tensorflow.org/system/image/body/1704/softmax-regression-scalargraph.png

http://api.tensorflow.org/system/image/body/1707/softmax-regression-scalarequation.png

http://api.tensorflow.org/system/image/body/1705/softmax-regression-vectorequation.png

scalar equation should be:
[W11 * X1 + W12 * X2 +  W13 * X3 + b1]
[W21 * X1 +W22 * X2 + W23 * X3 + b2]
[W31 * X1 + W32 * X2 + W33 * X3 + b3] 
```

**Comment by vrv** [comment]
Thanks for the report -- we'll update the image soon and let you know when this is fixed!

**Comment by vrv** [comment]
Fixed in f6d5a2caedd3982a1e6dcaa99bfafe0539527144 by colah.  Thanks!

**Comment by vrv** [comment]
(will take some time to propagate to the website though)
